geps/gep-2649/index.md

```diff
@@ -0,0 +1,733 @@
+  Policy MUST be the set of all distinct fields inside the relevant Policy objects,
+  as set out by the rules below.
+* For Policies that affect an object's existing fields, multiple instances of the
+  same Policy Kind affecting an object's fields MUST be evaluated as
+  though only a single Policy "wins" the right to affect each field. This operation
+  is performed on a _per-distinct-field_ basis.
+* Settings in `overrides` stanzas MUST win over the same setting in a `defaults`
+  stanza.
+* `overrides` settings operate in a "less specific beats more specific" fashion -
+  Policies attached _higher_ up the hierarchy beat the same type of Policy
+  attached further down the hierarchy.
+* `defaults` settings operate in a "more specific beats less specific" fashion -
+  Policies attached _lower down_ the hierarchy beat the same type of Policy
+  attached further _up_ the hierarchy.
+* For `defaults`, the _most specific_ value is the one _inside the object_ that
+  the Policy applies to; that is, if a Policy specifies a `default`, and an object
+  specifies a value, the _object's_ value wins.
+* Policies interact with the fields they are controlling in a "replace value"
+  fashion.
+    * For fields where the `value` is a scalar, (like a string or a number)
+      MUST have their value _replaced_ by the value in the Policy if it wins.
+      Notably, this means that a `default` will only ever replace an empty or unset
+      value in an object. Note also that, as in Go, the empty string value is _only_
+      a string of length 0, generally represented as `""`.
+    * For fields where the value is an object, the Policy should include the fields
+      in the object in its definition, so that the replacement can be performed
+      on each field inside the object as a simple field rather than the object as
+      a whole.
+    * For fields where the final value is non-scalar, but is not an _object_ with
+      fields of its own, the value MUST be entirely replaced, _not_ merged. This
+      means that lists of strings or lists of ints specified in a Policy overwrite
+      the empty list (in the case of a `default`) or any specified list (in the case
+      of an `override`). The same applies to `map[string]string` fields. An example
+      here would be a field that stores a map of annotations - specifying a Policy
+      that overrides annotations will mean that a final object specifying those
+      annotations will have its value _entirely replaced_ by an `override` setting.
+* In the case that two Policies of the same type specify different fields, then
+  _all_ of the specified fields MUST take effect on the affected object, using
+  the precedence rules given above.
+
+Examples to further illustrate these rules are given below.
+
+#### Supported Resources
+It is important to note that not every implementation will be able to support
+policy attachment to each resource described in the hierarchy above. When that
```

Comment:
I think this section should be nested under the Hierarchy header that comes later in the doc.

Reply:
Good suggestion, thanks.